users/postgres/init.go

```diff
@@ -18,8 +18,8 @@ import (
 // Connect creates a connection to the PostgreSQL instance and applies any
 // unapplied database migrations. A non-nil error is returned to indicate
 // failure.
-func Connect(host, port, name, user, pass, sslMode string) (*sql.DB, error) {
-	url := fmt.Sprintf("host=%s port=%s user=%s dbname=%s password=%s sslmode=%s", host, port, user, name, pass, sslMode)
+func Connect(host, port, name, user, pass, sslMode, sslCert, sslKey, sslRootCert string) (*sql.DB, error) {
```

Comment:
I think that it makes sense to create db config structure and use it as parameter here.

Reply:
@anovakovic01 what do you mean by "db config structure"? I hope you are not thinking of storing the config in some db table ;)
If we need config lets do it as @srados suggested in #435

Reply:
@chombium I think he meant that instead of passing a set of parameters, he should pass a struct containing all parameters

Reply:
@chombium yes, I meant passing struct instead of all these parameters.

Reply:
@chombium, @srados Thanks for the clarification. I thought that too, but that db caught my eyes :)